seed/data_importer/match.py

```diff
@@ -272,13 +276,16 @@ def inclusive_match_and_merge(unmatched_state_ids, org, StateClass):
 
             promoted_ids.append(merge_state.id)
 
+        if batch_size > 0 and idx % batch_size == 0 and progress_data:
```

Comment:
Too many conditions? If there are no matched_id groups, batch size will be zero, resulting in a zeroDivisionError and proress_data is an optional arg.